src/Domain/CoordinateMaps/BulgedCube.cpp

```diff
@@ -157,6 +146,16 @@ boost::optional<std::array<double, 3>> BulgedCube::inverse(
   const double y_sq = square(physical_y);
   const double z_sq = square(physical_z);
   const double physical_r_squared = x_sq + y_sq + z_sq;
+  // There is no 'else' covering the case physical_r_squared==0.
+  // For physical_r_squared==0 we know that we
+  // are at the origin x=y=z=0, and we know analytically that this
+  // map maps the origin to itself. Handling the case r==0 case separately
+  // simplifies the root finding procedure for the scaling_factor.
+  if (UNLIKELY(physical_r_squared == 0.0)) {
+    return {{{0.0, 0.0, 0.0}}};
+  }
+
+  // We are not at the origin, fund the scaling factor (does a root-find)
```

Comment:
`fund`->`find`

Reply:
Squashed and pushed. Thanks for catching this!